README.md

```diff
@@ -1 +1,181 @@
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- It gives information about twenties leadership summit 2023.
+- It showcases past summits.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://seyioyemade.github.io/microverse-module-1-capstone-project/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+You need to have npm installed on your PC first.
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+[https://github.com/seyioyemade/microverse-module-1-capstone-project]
+
+
+
+### Install
+
+Install this project with:
+
+-Copy the URL [https://github.com/seyioyemade/microverse-module-1-capstone-project.git].
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+git clone https://github.com/seyioyemade/microverse-module-1-capstone-project.git
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+- GitHub Page
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Seyi Oyemade**
+
+- GitHub: [@seyioyemade](https://github.com/seyioyemade)
+- Twitter: [@seyi_oyemade](https://twitter.com/@seyi_oyemade)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/seyi-oyemade-523ab1142/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Develop registration page
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/seyioyemade/microverse-module-1-capstone-project/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project and would love to show your support, please contact the author.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank [Microverse](https://www.microverse.org/) for giving this opportunity to learn.
+- I would like to thank [Cindy Shin](https://www.behance.net/adagio07), the author of the original design.
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also hyperlink the `author the original design.` text to help users if they want to checkout they can easily find it. You can update it to something like:
```
  - I would like to thank [Cindy Shin](https://www.behance.net/adagio07), the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 
```

Reply:
@tresorsawasawa 

Thanks for the feedback. I will do that.